### ISSUE TITLE
Avoiding strings like "1 file(s)" in GUIs

### ISSUE THREAD

**Issue by tsujan** [lxqt/lxqt#1642]
In translations like `tr("%1 file(s)", "", N)`, all languages are happy except for the default language, namely, US En. You can find ugly strings like "1 item(s)" or "11 item(s)" in many Qt apps (on pcmanfm-qt's statusbar, for example).

I usually try to avoid strings like "file(s)" in the old-fashion way, i.e. by separating 0, 1 and N>1. As @Fat-Zer showed [here](https://github.com/lxqt/libfm-qt/pull/334), N>1 should be written as `tr("%1 files", "", N)` -- and not `tr("%1 files").arg(N)` -- for all translations to be correct and for `(s)` to be avoided in US En too.

However, @Fat-Zer also had a good point [here](https://github.com/lxqt/libfm-qt/pull/333) as to why separating 0, 1 and N>1 from each other (my way) may not be a good solution.

Instead, `lupdate` has a `-pluralonly` option that can be used for making US En translations only for plural forms. Then, `tr("%1 item(s)", "", N)` can appear correctly, as "1 item" or "11 items".

In short, to each LXQt project that contains plural form translations like `"%1 file(s)`, a small US En transition file, that only covers such strings, can be added.

I don't know whether automatic updating of these small translations would be easy because they should be updated differently.

**Comment by Fat-Zer** [comment]
Wow, I really haven't expected to see such a proposition so soon! To be honest I thought that's something that would never be done. =)

The `-pluralonly` option sounds especially great: it should be really easy to integrate into both: CMake scripts and Weblate (it supports different source files for different languages, isn't it?). And it should bring virtually no confusion like giant *.po «you should not translate» or long obscure instructions, «how to translate a single 10-character line from English to English».

PS: To avoid mistakes, it's actually `tr("%n files", "", N)`. `tr` uses `%n` (literally) notation.
PPS: Sorry, I was a bit harsh on the discussion of the first pull request.
